subprojects/gradle/instrumentation-tests/src/test/kotlin/com/avito/instrumentation/TestRunResultJsonTest.kt

```diff
@@ -0,0 +1,47 @@
+package com.avito.instrumentation
+
+import com.avito.instrumentation.internal.InstrumentationTestsActionFactory.Companion.gson
+import com.avito.instrumentation.internal.TestRunResult
+import com.avito.instrumentation.internal.TestRunResult.Verdict.Failure.Details
+import com.github.salomonbrys.kotson.fromJson
+import com.google.common.truth.Truth
+import org.junit.jupiter.api.Test
+
+class TestRunResultJsonTest {
+    @Test
+    fun `verdict is success`() {
+        val expected: TestRunResult.Verdict = TestRunResult.Verdict.Success("Success")
+        val json = gson.toJson(expected)
+        val actual = gson.fromJson<TestRunResult.Verdict>(json)
+
+        Truth.assertThat(actual)
```

Comment:
seems you don't like static imports of assertThat; it's used in whole project instead of this test

Reply:
I don't care about it but there is no problem importing this

Reply:
i care only for consistency